### PR TITLE
Fixed generators to better account for the lack of return value of clSVMFree.

### PR DESCRIPTION
**Pull request by Kerilk**
As raised by https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/115.

### CODE REVIEW

File: scripts/icd_dispatch_generated.c.mako [hunk 1]
```diff
@@ -78,14 +78,23 @@ ${("", "static ")[disp]}CL_API_ENTRY ${api.RetType} CL_API_CALL ${api.Name + (""
 %if disp == 0:
 #if defined(CL_ENABLE_LAYERS)
     if (khrFirstLayer)
+%  if api.Name == "clSVMFree":
```
Comment (bashbaug) [inline scripts/icd_dispatch_generated.c.mako:81]: This is fine for now, but if we wanted to make this a little more general we could check for a `void` return type here vs. `clSVMFree` specifically.

**Review by bashbaug — COMMENTED** [review]
Thanks for fixing this!

I think we'll need to generate new files now also?  Otherwise LGTM.

**Comment by Kerilk** [comment]
The fixed generated files are already in #115. I would like to avoid a conflict. What do you think?

File: scripts/icd_dispatch_generated.c.mako (hunk 1 above)
Comment (Kerilk) [reply scripts/icd_dispatch_generated.c.mako:81]: Yes it occurred to me, but I would have had to make this change several times throughout the file, so I kept it uniform. We will need to refactor it at some point.

**Comment by bashbaug** [comment]
I think we should try to keep the generated file in sync with the template, in which case the generated file should be included in this PR also.  I'll work on adding some automation to enforce this.

**Comment by Kerilk** [comment]
I updated the generated files. Thanks for the feedback Ben.

**Review by bashbaug — APPROVED** [review]
Thanks - LGTM.  Merging ASAP to unblock #115.

**Pull request merged**